dev/build.sh

```diff
@@ -70,6 +73,7 @@ DEST=$2
 
 if [[ -n "$DEST" ]]; then
     sudo install -m 754 -o dcrdata -g decred ./v3 ${DEST}/dcrdata
```

Comment:
Since all the previous binary versions including `v3` have been deleted, you could use a regex expression to select the latest executable binary built. i.e.
```shell
sudo install -m 754 -o dcrdata -g decred ./v[0-9]* ${DEST}/dcrdata
```